docs/concatenation.rst

```diff
@@ -26,7 +26,7 @@ Let's start off with an example:
     AnnData object with n_obs × n_vars = 700 × 765
         obs: 'bulk_labels', 'n_genes', 'percent_mito', 'n_counts', 'S_score', 'G2M_score', 'phase', 'louvain'
         var: 'n_counts', 'means', 'dispersions', 'dispersions_norm', 'highly_variable'
-        uns: 'bulk_labels_colors', 'louvain', 'louvain_colors', 'neighbors', 'pca', 'rank_genes_groups'
+        uns: 'bulk_labels_categories', 'bulk_labels_colors', 'louvain', 'louvain_categories', 'louvain_colors', 'neighbors', 'pca', 'phase_categories', 'rank_genes_groups'
```

Comment:
Similarly, this actually does seem correct unless I am missing something - when I print out the discovered keys from `uns`, it includes these at the top level.